src/borg/helpers/parseformat.py

```diff
@@ -405,6 +414,16 @@ def normpath_special(p):
             p = os.path.normpath(p)
             return ('/.' + p) if relative else p
 
+        m = self.win_file_re.match(text)
+        if m:
+            self.proto = 'file'
+            self.path = os.path.normpath(m.group('path'))
+            self.archive = m.group('archive')
+            # Only consider the parsing complete if the path is not the current directory.
+            return self.path != '.'
+        if is_win32:
+            # On windows only windows path are supported
```

Comment:
is that rather: "on windows, we **currently** only support windows paths"?

Reply:
or even: ... we do not currently support anything else than windows paths.

Reply:
Fixed